cmd/config/api/api.go

```diff
@@ -78,6 +80,10 @@ var (
 			Key:   apiListQuorum,
 			Value: "optimal",
 		},
+		config.KV{
+			Key:   apiExtendListCacheLife,
+			Value: "0s",
```

Comment:
is default value 0s?

Reply:
Yes, it keeps the current behavior. Which requires the incoming listing to have come in before any set has finished listing.